### ISSUE TITLE
Bug with bullet navigation

### ISSUE THREAD

**Issue by SixStringSamurai** [zurb/orbit#22]
I have a page with two instances of the slideshow running.  Each instance displays two pictures.  

The first slideshow shows 4 bullets (two extraneous).  The first two of work as they should, with the "selected" bullet changing as the displayed picture changes.  The second two bullets just appear for no reason, and never become "selected". 

The second slideshow shows 2 bullets, but neither ever appear as "selected," no matter which slide is being displayed.

**Comment by lukemcdonald** [comment]
I'm experiencing the same issue using version 1.2.3.

**Comment by vasilpe** [comment]
I'm also having the same issue in version 1.2.3. 

We have a page with four slideshows, where the first and the fourth slider are meant to have a bullet navigation on them. What happens in this case is, that the slider number 1, that has 3 slides, displays with 5 bullets, and the slider number 4, that has 2 slides, displays with the correct number of bullets, but without displaying which bullet is active.

**Comment by RazvanGirmacea** [comment]
Same problem. You can see it on http://girmacea.ro/

**Comment by intelligence** [comment]
I have the same issue aswell

Here's a demo:
http://persturesson.com/orbit/

**Comment by mkelly12** [comment]
This was a problem with the minified version of the code not being updated. I removed the minified version from GitHub so we don't end up with outdated code.

I also updated the full package with the latest code (that includes this fix) if you still want a minified version:
http://matt.zurb.s3.amazonaws.com/orbit-1.2.3.zip

Going forward the latest code will always be available on GitHub, and we will release full packages on the playground page that will include the minified version.
